Application.Models/Response/CustomerResponse.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Application.Models.Response
+{
+    public class CustomerResponse
+    {
+        public string Fullname { get; set; }
```

Comment:
FullName são 2 palavras, logo pascal case https://en.wiktionary.org/wiki/full_name
```suggestion
        public string FullName { get; set; }
```